client/src/Components/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAngleUp, faAngleDown } from '@fortawesome/free-solid-svg-icons';
+
+interface CollapsibleProps {
+  title: string;
+  children: React.ReactNode;
+}
+
+const Collapsible: React.FC<CollapsibleProps> = ({ title, children, ...rest }) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggleCollapsible = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <div className="border rounded-md shadow-md" {...rest}>
+      <div className="flex items-center justify-between cursor-pointer" onClick={toggleCollapsible}>
+        <h3 className="text-lg py-2 pl-3">{title}</h3>
+        {isOpen ? <FontAwesomeIcon icon={faAngleUp} /> : <FontAwesomeIcon icon={faAngleDown} />}
```

Comment:
I am not a big fan of font awsome because it added a lot of weight to the website I would normally go to something that give us a bit more flexibility even to colour and it is compatible with css 
https://github.com/tailwindlabs/heroicons#react